### PR TITLE
K8SPXC-674 RedHat certification files

### PR DESCRIPTION
**Pull request by cap1984**
[K8SPXC-674](https://jira.percona.com/browse/K8SPXC-674) [Powered by Pull Request Badge](https://pullrequestbadge.com/?utm_medium=github&utm_source=percona&utm_campaign=badge_info)

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| haproxy | passed |
| storage | passed |
| scheduled-backup | passed |
| big-data | passed |
| scaling | passed |
| upgrade-haproxy | passed |
| recreate | passed |
| init-deploy | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| scaling-proxysql | passed |
| upgrade-proxysql | passed |
| limits | passed |
| security-context | passed |
| self-healing-advanced | passed |
| monitoring-2-0 | failed |
| smart-update | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| pitr | passed |
| proxysql-sidecar-res-limits | failed |
| upgrade-consistency | passed |
| demand-backup-encrypted-with-tls | passed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/870/commits/841433e85d1f4db1690f75baa36e22f683377c4e
image: `perconalab/percona-xtradb-cluster-operator:PR-870-841433e8`

File: deploy/csv/redhat/bundle-1.8.0.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.2.0
+LABEL operators.operatorframework.io.metrics.mediatype.v1=metrics+v1
+LABEL operators.operatorframework.io.metrics.project_layout=helm.sdk.operatorframework.io/v1
+LABEL operators.operatorframework.io.test.config.v1=tests/scorecard/
+LABEL operators.operatorframework.io.test.mediatype.v1=scorecard+v1
+
+#Add these labels
+LABEL com.redhat.openshift.versions="v4.5,v4.6"
```
Comment (tplavcic) [inline deploy/csv/redhat/bundle-1.8.0.Dockerfile:16]: add `v4.7`?

**Review by tplavcic — CHANGES_REQUESTED** [review]

File: deploy/csv/redhat/bundle-1.8.0.Dockerfile (hunk 1 above)
Comment (cap1984) [reply deploy/csv/redhat/bundle-1.8.0.Dockerfile:16]: no mention here https://redhat-connect.gitbook.io/certified-operator-guide/appendix/bundle-maintenance-after-migration

**Review by hors — APPROVED** [review]

File: deploy/csv/redhat/bundle-1.8.0.Dockerfile (hunk 1 above)
Comment (tplavcic) [reply deploy/csv/redhat/bundle-1.8.0.Dockerfile:16]: what does that mean? from what I see the tutorial is old and was created before 4.7 was released, what is the purpose of this LABEL field?

**Review by tplavcic — CHANGES_REQUESTED** [review]

**Review by tplavcic — APPROVED** [review]

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| scaling | passed |
| scaling-proxysql | passed |
| security-context | passed |
| haproxy | passed |
| init-deploy | passed |
| limits | passed |
| monitoring-2-0 | failed |
| storage | passed |
| recreate | passed |
| self-healing | passed |
| restore-to-encrypted-cluster | passed |
| self-healing-advanced | passed |
| demand-backup | passed |
| operator-self-healing | passed |
| pitr | passed |
| demand-backup-encrypted-with-tls | passed |
| scheduled-backup | passed |
| big-data | passed |
| upgrade-haproxy | passed |
| upgrade-proxysql | passed |
| smart-update | passed |
| upgrade-consistency | passed |
| affinity | passed |
| one-pod | passed |
| auto-tuning | passed |
| proxysql-sidecar-res-limits | passed |
| users | passed |
| tls-issue-self | passed |
| tls-issue-cert-manager | failed |
| tls-issue-cert-manager-ref | passed |
| validation-hook | failed |

commit: https://github.com/percona/percona-xtradb-cluster-operator/pull/870/commits/841433e85d1f4db1690f75baa36e22f683377c4e
image: `perconalab/percona-xtradb-cluster-operator:PR-870-841433e8`

**Pull request merged**